opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         self.value_type = value_type
         self.enabled = enabled
         self.monotonic = monotonic
+        self.time_stamp = datetime.utcnow()
```

Comment:
You want to use https://github.com/open-telemetry/opentelemetry-python/blob/764d3171d7435915bc536da184d6ba44b241581f/opentelemetry-sdk/src/opentelemetry/sdk/util.py#L30.